src/functions/routeHandler.ts

```diff
@@ -0,0 +1,25 @@
+import * as express from 'express'
+import * as next from 'next'
+
+import { isHostedOnAWS } from '../utils/aws'
+
+const port = process.env.PORT ? process.env.PORT : '3000'
+const nextOptions = { dev: !isHostedOnAWS }
+const app = next(nextOptions)
+const handle = app.getRequestHandler()
+
+function createServer(req: any, res: any) {
+	console.log(`req from ${req.headers.host}`)
+	handle(req, res)
+}
+
+if (!isHostedOnAWS) {
+	app.prepare().then(() => {
+		const server = express()
+		server.get('*', (req, res) => handle(req, res))
+		server.listen(3000)
```

Comment:
Hard coded port

Reply:
Thanks! I forgot to use the `port` variable here.